talentlink-connector/src/com/axonivy/connector/talentlink/features/RestApiFeature.java

```diff
@@ -0,0 +1,14 @@
+package com.axonivy.connector.talentlink.features;
+
+import javax.ws.rs.core.FeatureContext;
+
+import ch.ivyteam.ivy.rest.client.mapper.JsonFeature;
+
+public class RestApiFeature extends JsonFeature{
```

Comment:
pretty generic name, better would be something like:
- TalentLinkAuthFeature
- TalentLinkApiKeyFeature
- ...